WaterbendingToph/cs414-f20-public_class_TeamD
# [Fixes #129] Connect Database to Login Controller

TO TEST:
- SSH to CS lab machine
- Load back end server and localhost:8080
- Attempt login with existing user/password in database (e.g. 'nick', '42') and see if it redirects you to the lobby
- Attempt login with bad user/password not in database and ensure it doesn't send you anywhere (just reloads login page for now)
- Output in server terminal/console should show the userID, password, date, and whether or not you've successfully logged in, as well as the response from the database (which will be either the username or '')
